src/Engine/ProtoCore/FFI/FFIExecutionManager.cs

```diff
@@ -15,7 +15,7 @@ private FFIExecutionManager()
         }
 
         Dictionary<RuntimeCore, FFIExecutionSession> mSessions = new Dictionary<RuntimeCore, FFIExecutionSession>();
-        ExtensionAppLoader mApploader = new ExtensionAppLoader();
+        ExtensionAppLoader mApploader = ExtensionAppLoader.Instance;
```

Comment:
No, although it is singleton, but it isn't used like a singleton. When core is disposed, `FFIExecutionManager` will set `mSelf` to null and re-create one later on.